panther_battery/README.md

```diff
@@ -19,9 +19,73 @@ Publishes battery state read from ADC unit. Available from Panther version 1.2.
 - `/panther/driver/motor_controllers_state` [*panther_msgs/DriverState*]: current motor controllers' state and error flags.
 - `/panther/hardware/io_state` [*panther_msgs/IOState*]: checks if charger is connected. Later fuses the information with charging current.
 
-#### Parameters 
-
-- `~fatal_bat_temp` [*float*, default: **62.0**]: The temperature of the battery at which the battery health state is set to `POWER_SUPPLY_HEALTH_OVERHEAT`.
+#### Battery statuses
+
+The battery status is described with two [Battery State](http://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/BatteryState.html) message fields: `power_supply_status` and `power_supply_health`. **This information is subsequently retrieved in the panther_manager node, where decisions related to safety procedures are made**.
+
+Safety thresholds used in calculating the various values describing the state of the battery:
+
+| Name                       | Value | Unit       |
+| -------------------------- | ----- | ---------- |
+| `BAT_CHARGING_CURR_THRESH` | 0.1   | **A**      |
+| `V_BAT_FATAL_MIN`          | 27.0  | **V**      |
+| `V_BAT_FATAL_MAX`          | 43.0  | **V**      |
+| `V_BAT_FULL`               | 41.4  | **V**      |
+| `V_BAT_MIN`                | 32.0  | **V**      |
+| `LOW_BAT_TEMP`             | -10.0 | **&deg;C** |
+| `OVERHEAT_BAT_TEMP`        | 45.0  | **&deg;C** |
+
+The method of calculating individual values:
+
+- for single battery:
+
+| Value                                       | Variable name                                                 | Method of calculating                            | Comment                                                                                          | Published on |
+| ------------------------------------------- | ------------------------------------------------------------- | ------------------------------------------------ | ------------------------------------------------------------------------------------------------ | ------------ |
+| Voltage                                     | `V_bat`                                                       | Measured with ADC unit                           | -                                                                                                | `/battery`   |
+| Temperature                                 | `temp_bat`                                                    | Measured with thermistor                         | -                                                                                                | `/battery`   |
```

Comment:
The value measured is actually V_temp, later it is converted to temp value with `_voltage_to_deg` function.